roles/StackStorm.st2smoketests/tasks/st2chatops.yml

```diff
@@ -1,15 +1,41 @@
 ---
+- name: Prepare temp file to track retry attempts
+  tempfile:
+    state: file
+  register: attempts_count
```

Comment:
does this initialize attempts_count to 0 on the first run?

Reply:
I should rename that ansible var to attempts_count_file to mirror the var name I'm using in Bash and make the var easier to follow.
If the file is empty, it defaults to 0 because of the 0 before the cat here:
`n=0$(cat $attempts_count_file)`